### ISSUE TITLE
List of gateways and coins

### ISSUE THREAD

**Issue by froooze** [bitshares/bitshares.org#83]
I am missing a list of gateways and supported coins. It is hard to keep an overview and a lot of gateways have no list of supported coins on their website. 

To get listed on bitshares.org, the standards must be filled and this gives enough motivation to provide informations.   

Highlight exchanges, which provide also BTS pairings.

**Comment by rsswlkr** [comment]
It exists - Dashboard on wallet.bitshares.org has the list. You can even sort by liquidity / volume. Also under 'Legacy Deposit' - select a Gateway, view the available coins to deposit (you'd be wise to check their volumes/liquidity). These remain 2 of the ways to check what is supported.

![dex-dashboard1](https://user-images.githubusercontent.com/28939873/58930404-649e3200-8785-11e9-8553-38c989b70b27.png)

Could be out of scope as far as main BitShares.org site is concerned, biggest issue being the amazing amount of content to maintain there - you said yourself it is hard to keep an overview, well, listings are changing daily for hundreds of coins ... so yes really it is. (IMO) 

However, 2 good ideas out of this: 

-  potential a page somewhere to highlight the main gateways and businesses on BitShares - i will wait for others feedback first
- a 'Token Factory' page about UIA's is coming up. This has given me an idea to extend the concept some more whereby it describes the process as 'listing on BitShares' (can build few use cases around it too).

**Comment by rsswlkr** [comment]
Guess I had to read 2 times. As far as we know, Move Institute being located in a well regulated cryptocurrency jurisdiction (Slovenia), and we can't list literally everyone. Especially those that have no proper paperwork, company address, phone number to be called in case of regulators coming knocking to ask after them. 

In the terms of branding and backlinking, since there is an ongoing discussion to remove the blocktivity table from the homepage, maybe it would be nice to replace this with list of top well-known top regulated exchanges listing BTS, a la Binance, Bittrex etc and provide backlink to them. 

More details on legal stuff and exchange listings - hope @dls-cipher can help here

**Comment by dls-cipher** [comment]
Thanks @rsswlkr for explanation :)

@froooze - More milestones will be released to forums/github and publications will follow up when the time is right. 

I can say for now this much: 
Ethereum Alliance is similar to business directory of Ethereum chain. It was the panel/registration/kyc for respected business to get access to each other over ethereum chain + a lot of goodies in early days, such as token factory easy access, early erc20 adoption, etc. Not worldwide masses acting chaotic and hectic, but well known companies/brands. That how Ethereum survived regulations, by selling service + applying KYC on businesses and distancing by agreement during that process that client (business) is responsible for any doing on ethereum chain. 

To be able to list gateways/their UIA's without any proof of liquidity backing them, governments or banks to prove actual existence of anything they are trading/selling, we have to embed/create such process of registration (for business only). 

**So example:** OpenLedger comes and boards in with their legit paperwork and coins they wanna list (from legit perspective), pays annual membership and gets boarded in to the Business Directory of BitShares with all their spotlights, updates, direct support through bitshares.org (Move Institute) without us having any responsibility for their clients/traders/users. 

**Bottom line:** We can't just list and show entire planet all UIA's and gateways currently listed on BitShares. It's a long process before we become clear on definition of responsibility around ecosystem before we start pushing hard marketing on something that can mean end for all of us one day. Regulations are tighter and there are ways to "comply" where just gateways (who earns the most in the end of the day) have to step-up for responsibility and protect THEIR clients utilizing BitShares blockchain. Also, if want BitShares advertisement, pay to the BitShares reserve pool annual membership and lets help the blockchain to live longer. 

It's not that black & white/simple as your request is, and there is 5 months left in this worker to come to the point where we have proper departments, support, listings, legal, downloads, team and brand to show to the world, not just randomness that is popping up every year like it did in the past. 

Chee®s

**Comment by dls-cipher** [comment]
@froooze - Although, on the news.bitshares.org most of them will be listed, all their dapps will be listed and some of them will have author access to publish their important updates/articles that are BitShares related.

**Comment by xeroc** [comment]
Finally moving from an unmaintained hobby webpage to an actual business! +5%

**Comment by dls-cipher** [comment]
Discussion to be continued in #89 - closing this one.
